content/docs/studio/index.md

```diff
@@ -1,11 +1,12 @@
 # Iterative Studio
 
 [Iterative Studio](https://studio.iterative.ai/) is a web application that you
-can access online or even host on-prem. It works with the data, metrics, and
+can access online or even host on-prem. It works with the data, metrics and
```

Comment:
Unrelated change. `,` was correct?

Reply:
We don't follow this style everywhere, so removed it.